manifests/service.pp

```diff
@@ -21,23 +21,25 @@
     default => false,
   }
 
-  if $openntp::service_restart == $openntp::params::service_restart {
-    # The restart script does not properly check if the service is actually stopped; hence, the start command can fail
-    # to start the service because it is still running. This script checks the status and issues the start command as
-    # soon as the service is actually stopped.
+  if $openntp::service_restart != undef {
+    if $openntp::service_restart == $openntp::params::service_restart {
```

Comment:
I separated the two if blocks for other variants of openntp::service_restart to come, but the two blocks can certainly be merged.